package.json

```diff
@@ -50,7 +50,8 @@
     "loader.js": "^4.2.3"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "^4.5 || 6.* || >= 7.* < 10.*",
```

Comment:
Let's constrain this to be node v7 or higher - do we need the yarn specification?

Reply:
We saw issues from not tying the yarn version down in account and we need to specify `< 10` for ember-cli < v3